src/sapi.c

```diff
@@ -18,6 +18,14 @@ void defineClassNativeField(VM* vm, const char* id, Constant field, GCClass* cla
   pop(vm);
 }
 
+void defineGlobalFunc(VM* vm, const char* id, NativeFunc function) {
+  push(vm, GC_OBJ_CONST(copyString(vm, NULL, id, (int)strlen(id))));
+  push(vm, GC_OBJ_CONST(newNative(vm, function)));
+  tableInsert(vm, &vm->globals, AS_STRING(vm->stack[0]), vm->stack[1]);
+  pop(vm);
+  pop(vm);
+}
+
```

Comment:
This function is declared below defineGlobal in the header file, please keep the same order